ethereum/core/src/main/java/org/hyperledger/besu/ethereum/chain/GenesisState.java

```diff
@@ -220,7 +217,7 @@ private static BlockHeader buildHeader(
         .excessBlobGas(isCancunAtGenesis(genesis) ? parseExcessBlobGas(genesis) : null)
         .parentBeaconBlockRoot(
             (isCancunAtGenesis(genesis) ? parseParentBeaconBlockRoot(genesis) : null))
-        .requestsRoot(isPragueAtGenesis(genesis) ? Hash.EMPTY_TRIE_HASH : null)
+        .requestsHash(isPragueAtGenesis(genesis) ? Hash.EMPTY_TRIE_HASH : null)
```

Comment:
Wondering if we should be using `Hash.EMPTY_TRIE_HASH` here yet

Reply:
Yeah this is wrong, I've updated to do the requests hash of the empty requests i.e:

```
hash = keccac256(keccac256(0x00) ++ keccac256(0x01) ++ keccac256(0x02))
```